.travis.yml

```diff
@@ -19,6 +19,7 @@ before_script:
 env:
   global:
     - PGUSER=postgres
+    - PG_SOCKET_DIR=/var/run/postgresql
```

Comment:
We should instead set this in `test_helper.exs`, otherwise every developer have to set this every time they run tests. Maybe set to `PROJECT_DIR/tmp` and gitignore it?

Reply:
The problem is that different distributions put the socket on different places. I think the best is to attempt to detect some known locations in test_helpers and skip the tests if we can't find one.

Reply:
We should probably pick this as default then and document how developers can override it.